<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1685</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>1121</width>
      <height>761</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
    <widget class="QGraphicsView" name="mapView"/>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QPushButton" name="addSectorButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir sector</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addWallButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir pared</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTextureButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir textura</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exportar mapa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sectores :</string>
      </property>
     </widget>
     <widget class="QListWidget" name="sectorList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Altura de piso : </string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="floorHeightSpin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ceilingHeightSpin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>62</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-100000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Altura de techo :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>58</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Texturas : </string>
      </property>
     </widget>
     <widget class="QListWidget" name="textureList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>341</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="editVerticesButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Editar vertices</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteSectorButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar sector</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Textura de suelo :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Textura de techo :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Textura de pared :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="floorTextureCombo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>470</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="ceilingTextureCombo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>500</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="wallTextureCombo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>530</y>
        <width>79</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="importButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Importar mapa</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1685</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
